docs/backups.rst

```diff
@@ -45,10 +45,14 @@ Drop your existing local database and restore from the backup::
 There will be a bunch of errors related to the fact that the production dump has roles which your
 local DB doesn't have. It's OK to ignore them.
 
-Migrate and update the CMS 'site' so that your local server uses production content::
+Migrate the database::
 
     $ workon serviceinfo
     $ ./manage.py migrate --noinput
+
+Next, we need to copy the production CMS content to your local site. The value 'localhost:8000' is
```

Comment:
If this is analogous to WordPress, I'm guessing this isn't really a copy, just updating some places in the data that know the domain name of the site that we backed up the data from, so they'll work on a site with a different domain name.

Reply:
I honestly don't know what this command does. Let me know what wording you'd like.

Reply:
OK, I've spent more time digging into the code, and I do think that this actually copies pages from one site to another: https://github.com/theirc/ServiceInfo/blob/develop/service_info_cms/utils.py#L44-L46

Reply:
Maybe something like "Next, we need to update the site name in the CMS data."?

Reply:
Updated.